src/main/java/edu/hillel/homework/logger_system/log4hillel/config_loader/DatabaseLoggerConfigLoader.java

```diff
@@ -0,0 +1,62 @@
+package edu.hillel.homework.logger_system.log4hillel.config_loader;
+
+import edu.hillel.homework.logger_system.log4hillel.LogLevel;
+import edu.hillel.homework.logger_system.log4hillel.core.configuration.DatabaseLoggerConfiguration;
+import edu.hillel.homework.logger_system.log4hillel.core.configuration.LoggerConfigurationInterface;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class DatabaseLoggerConfigLoader implements LoggerConfigLoaderInterface {
+
+    private final String configFilePath;
+
+    public DatabaseLoggerConfigLoader(String configFilePath) {
+        this.configFilePath = configFilePath;
+    }
+
+    @Override
+    public LoggerConfigurationInterface load() {
+        try (BufferedReader r = new BufferedReader(new FileReader(configFilePath))) {
+            boolean databaseLoggingEnable = false;
+            LogLevel logLevel = null;
+            String format = null;
+
+            String line;
+            while ((line = r.readLine()) != null) {
+                if (line.contains("##")) {
+                    continue;
+                }
+                String[] parts = line.split("=");
+                if (parts.length != 2) {
+                    continue;
+                }
+
+                String key = parts[0].trim().toUpperCase();
+                String value = parts[1].trim();
+
+                switch (key) {
+                    case "DATABASE-LOGGING-ENABLED" -> {
+                        try {
+                            databaseLoggingEnable = Boolean.parseBoolean(value);
+                        } catch (Exception e) {
```

Comment:
The same here and you will avoid a code duplication

Reply:
Well done.